index.html

```diff
@@ -28,19 +28,192 @@ <h1 class="introduction">I'm Emmanuel, glad to see you</h1>
 
         <p class="description">I’m a software developer! I can help you build a product , feature or website Look 
             through some of my work and experience! If you like what you see and have a project you need coded, don’t 
-            hestiate to contact me.</p>
-    
+            hestiate to contact me.</p>    
         <h4 class="connect">LETS CONNECT</h4>
+        <ul class="social">
+            <li><a href="#"><img src="images/twit.png" alt="Twitter"></a></li>
+            <li><a href="#"><img src="images/Vector.png" alt="Linkedin"><img src="images/in.png" alt=""></a></li>
+            <li><a href="#"><img src="images/m.png" alt="Mi"></a></li>
+            <li><a href="#"><img src="images/cat.png" alt="Guthub"></a></li>
+            <li><a href="#"><img src="images/peace.png" alt="Alison"></a></li>
+        </ul>
+    </section>
+    <!--Welcome page end-->
+
+    <!--My projects start-->
+    <div class="my-works">
+        <figure>
+            <img src="images/tonic-project.jpg" class="works-image" alt="Tonic App" width="100%"> 
+        </figure>
+        <div class="works-description">
+            <h3 class="my-works-project-title">Tonic</h3>
+            <ul class="works-headers">
+                <li class="title-header">CANOPY</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">Back End Dev</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">2015</li>
+            </ul>
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <section class="project-languages">
+            <div class="coding-languages">html</div>
+            <div class="coding-languages">css</div>
+            <div class="coding-languages">javascript</div>
```

Comment:
Kindly adjust the styling (fonts, color, margins, etc) of the project languages to match your chosen design template. It'll be great to learn right now to stick to the given design as much as possible. This will get you prepared for a typical workplace experience where you might be required to produce websites according to the given design.

So, let's stick to the design, it'll be a great website.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192075811-cdbab472-fb68-4dff-9ec1-2cb89afb4b36.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/192075837-61ae43ce-29b0-4f39-b03f-5fe0aaeaa8b0.png)